dev/plugins/hu.elte.txtuml.export.javascript/src/hu/elte/txtuml/export/javascript/resources/res/pollingRefresh.js

```diff
@@ -88,3 +89,34 @@ $('#debug-port-input').on('keyup change input', function(){
 		typingTimer = setTimeout(doneTyping, DONE_TYPING_TIMEOUT_IN_MILLISECONDS);
 	}
 });
+
+//send animation delay to the server
+function sendSMTime(){
+	$.ajax({
+		url: 'http://localhost:' + port + '/delay',
+		type: 'POST',
+		dataType: 'json',
+		data: {delayTime:stateMachineSpeed},
+	    success: function(response) {
+        	if(response.status == 200 && isPolling){
+				setActiveElements((JSON.parse(response.responseText))
+					.map(entry => _visualizer.getShapeIdByElementName(entry.location)));
+				hideError();
+			}
+			else{
+				clearCurrentActiveElements();
+				if(isPolling){
+					showError();
+				}else{
+					hideError();
+				}
+			}
+    	}
+	});
+}
+
+//add event listeners to the state machine's speed range
+$('#animation-speed-input').on('click', function(){
```

Comment:
Change this to on `change`. 
This way it will
- work when the mouse button is released when not on the slider
- not trigger when the slider remained on the same position
- still only trigger when the mouse button is released, and not every time the position is changed